cmd/private-org-sync/main.go

```diff
@@ -124,9 +124,9 @@ func gitExec(logger *logrus.Entry, dir string, command ...string) (string, int,
 
 type RemoteBranchHeads map[string]string
 
-func getRemoteBranchHeads(logger *logrus.Entry, git gitFunc, repoDir, remote string) (RemoteBranchHeads, error) {
+func getRemoteBranchHeads(logger *logrus.Entry, git gitFunc, remote string) (RemoteBranchHeads, error) {
 	heads := RemoteBranchHeads{}
-	out, exitCode, err := git(logger, repoDir, "ls-remote", "--heads", remote)
+	out, exitCode, err := git(logger, "", "ls-remote", "--heads", remote)
```

Comment:
Instead of passing `""` I'd rather we had a `func git()` analogue that did not need a dir